src/dtos/view-dto.ts

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable import/no-cycle */
+import { Error } from '../models/error';
+
+import { DatasetDTO } from './dataset-dto';
+
+export interface PageInfo {
+    total_records: number | undefined;
+    start_record: number | undefined;
+    end_record: number | undefined;
+}
```

Comment:
So this forms the paging information for returning very big CSVs... I should probably use `start_line`, `end_line` and `total_lines` rather than records.  I can change this later if we think it is an important distinction 🤔